### ISSUE TITLE
negative reward notice but positive reward

### ISSUE THREAD

**Issue by meswrex** [Rocologo/MobHunting#97]
Hi again :)
Found an issue where a penalty message appears (red text) when killing a mob (eg Skeleton) but a positive money reward is still correctly given. eg: `You lost 25.84 Emeralds!` but this amount added to the player balance (as desired). Just wanting to resolve the incorrect message.

Version: MobHunting-3.4.7-SNAPSHOT-20160907180800-B120
Steps to reproduce:
- delete config file
- restart server to allow MobHunting plugin to regenerate default config file
- change `world-difficulty-multiplier` from default value of 0.75 to 1
- change `drop-money-on-groud` from default value of false to true
- set game difficulty to 1 (easy)
- issue command `summon Skeleton`
- kill skeleton and observe penalty message

Hopefully this will reproduce the issue.

**Comment by Rocologo** [comment]
Thank you for all the details. I have tried to do exactly like you, but I cant replicate the bug??? 

I see you use Emeralds? Do you use the Gringots plugin too?

**Comment by Rocologo** [comment]
So I enabled Gringots on my test server too, but still no luck, I cant replicate the bug. The text message is green and gives the right message. 

I do not doubt that you can replicate the bug, but I cant fix it if I cant replicate it too. :-( So I need your help to find the root cause and when this happens.

**Comment by Rocologo** [comment]
So I did something else....  I looked in the source code and examined the code where I write the message with red and then I think I discovered the bug. 

The error only happened when the sum of multipliers was exactly 1 - this was not the case on my test server no matter using your configuration. (I had a rank multiplier as well)

So I made a small change to fix this. Try build 122 or newer.

http://jenkins.lindegaard.one:8180/job/MobHunting/lastSuccessfulBuild/
